packages/generators/app/lib/create-project.js

```diff
@@ -81,7 +81,7 @@ module.exports = async function createProject(scope, { client, connection, depen
 
     if (useTypescript) {
       const tsJSONDir = join(__dirname, 'resources', 'json', 'ts');
-      const files = ['tsconfig-server.json.js', 'tslint.json.js'];
+      const files = ['tsconfig-admin.json.js', 'tsconfig-server.json.js', 'tslint.json.js'];
```

Comment:
Can you remove the tslint.json from this list & from the files directory? tslint has been deprecated in favour of an eslint default support. We'll configure it in another PR